Content.Shared/Hands/EntitySystems/SharedHandsSystem.Drop.cs

```diff
@@ -134,8 +134,10 @@ public bool TryDrop(EntityUid uid, Hand hand, EntityCoordinates? targetDropLocat
             return true;
         }
 
+        var origin = TransformSystem.GetMapCoordinates(entity);
         var target = targetDropLocation.Value.ToMap(EntityManager, TransformSystem);
-        TransformSystem.SetWorldPosition(itemXform, GetFinalDropCoordinates(uid, userXform.MapPosition, target));
+        var itemRot = TransformSystem.GetWorldRotation(entity); // based on camera orientation... 
```

Comment:
You should never be calling these separately, use the combined methods.

Reply:
in this scenario, GetMapCoordinates returns a type of MapCoordinates which is required for GetFinalDropCoordinates

If I were to use GetWorldPositionRotation, it returns a type of Vector2 instead of MapCoordinates

Reply:
so it's not quite a drop in-fix... I'd have to alter GetFinalDropCoordinates to use Vector2 positions instead of taking MapCoordinates? is that preferred...?

Reply:
I implemented it in a way using the target's MapId that ends up working but I'm not happy with how I did it? I would prefer a recommendation for how to do it a bit less scuffed.

Reply:
I thiiiiink I did it?